app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RecipeIngredientsController < ApplicationController
+  def update
+    redirect = Recipe.find(params[:id])
+    recipe = Recipe.create!(name: params[:name], cost: params[:cost])
+    RecipeIngredient.create!(recipe_id: redirect.id, ingredient_id: recipe.id)
+
+    redirect_to "/recipes/#{recipe.id}"
+    recipe_ingredients = RecipeIngredient.find(@recipe.id)
```

Comment:
Since we're redirecting to a recipe show page, this line will not be used or read.